### ISSUE TITLE
Dataset with no speaker name

### ISSUE THREAD

**Issue by huynq2k4** [nguyenhoanganh2002/XTTSv2-Finetuning-for-New-Languages#8]
I want to fine-tune XTTSv2 using the VietBud500 dataset, which contains only audio and transcription data, but lacks information about the speaker's name. Can the model be fine-tuned without specifying speaker names, and if speaker information is required, can I label each audio as a different speaker?

**Comment by nguyenhoanganh2002** [comment]
> I want to fine-tune XTTSv2 using the VietBud500 dataset, which contains only audio and transcription data, but lacks information about the speaker's name. Can the model be fine-tuned without specifying speaker names, and if speaker information is required, can I label each audio as a different speaker?

you can absolutely fine-tune XTTSv2 without needing `speaker_name` column in metadata files
